src/routes/controllers/subjects.controller.ts

```diff
@@ -43,9 +45,8 @@ export const addSubject = async (req: Request, res: Response) => {
   try {
     // manca la validazione del body
     const { name, hours } = req.body;
-    const subj: Subject = CreateSubject({ name, hours });
-    const result: Subject = await add(subj);
-    return res.status(201).json(result);
+    const subject: Subject = await add(name, hours);
```

Comment:
metti add({name, hours}a as Subject)

Reply:
Il metodo in ingresso richiamato richiede due stringhe in ingresso, e dato che non passiamo un oggetto, non possiamo utilizzare l'as Subject. Comunque sto notando che hai una versione vecchia del codice. Sicuro di essere nella branch corretta? e di essere aggiornato con tutti i commit?